app/main.rb

```diff
@@ -1,5 +1,5 @@
 # Require all of the Zif library:
-require 'app/lib/zif/require.rb'
+require 'lib/require.rb'
```

Comment:
@kfischer-okarin Will this work?

Reply:
Not Kevin, but yes this will work just fine 😄